modules/rostests/win32/user32/serverside/serverside.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * PROJECT:         ReactOS test suite
+ * LICENSE:         GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:         Test for existense of the kernel mode wndproc in the window
+ * COPYRIGHT:       Copyright 2024 Oleg Dubinskiy (oleg.dubinskiy@reactos.org)
+ */
+
+/* This testapp tests behaviour of IsServerSideWindow() function in user32. */
+
+#define _WINE
```

Comment:
This is not a wine test, so please remove this.

Reply:
Yeah, it's not. But this define is required to properly include undocuser.h (which contains `IsServerSideWindow` declaration) from winuser.h. See  But I think it can be included directly in apitest instead.

Reply:
No, this is a hack for Wine code only.
For ReactOS code, you explicitly include the undocuser.h header when you need it, see for example:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=undocuser.h